hazelcast-sql/src/test/java/com/hazelcast/sql/impl/plan/cache/PlanCacheIntegrationTest.java

```diff
@@ -1,218 +0,0 @@
-/*
- * Copyright 2021 Hazelcast Inc.
- *
- * Licensed under the Hazelcast Community License (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://hazelcast.com/hazelcast-community-license
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.hazelcast.sql.impl.plan.cache;
-
-import com.hazelcast.client.test.TestHazelcastFactory;
-import com.hazelcast.config.IndexType;
-import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.map.IMap;
-import com.hazelcast.sql.SqlResult;
-import com.hazelcast.sql.SqlRow;
-import com.hazelcast.sql.impl.QueryException;
-import com.hazelcast.sql.impl.SqlResultImpl;
-import com.hazelcast.sql.impl.exec.FaultyExec;
-import com.hazelcast.sql.impl.exec.scan.MapScanExec;
-import com.hazelcast.sql.impl.plan.Plan;
-import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.annotation.ParallelJVMTest;
-import com.hazelcast.test.annotation.QuickTest;
-import org.junit.After;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotSame;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.fail;
-
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
-public class PlanCacheIntegrationTest extends PlanCacheTestSupport {
```

Comment:
I would restore these tests.

Reply:
I restored only one test since other tests are redundant or already exists in other test classes.

Reply:
This test was verifying auto-discovered maps plan caching, i don't think we have other tests checking this.

Reply:
Restored.